src/Quizzer.API.BL/Facades/FacadeBase.cs

```diff
@@ -19,16 +19,25 @@ public abstract class FacadeBase<TEntity, TDetailModel> : IFacade<TEntity, TDeta
 
         protected virtual string[] IncludesNavigationPathDetails => Array.Empty<string>();
 
+        public virtual bool Exists(Guid Id) {
+            using IUnitOfWork uow = p_unitOfWorkFactory.Create();
+
+            using IRepository<TEntity> repo = uow.GetRepository<TEntity>();
 
-        public virtual TDetailModel GetDetailById(Guid id) {
+            return repo.Exists(Id);
+        }
+
+        public virtual TDetailModel? GetDetailById(Guid id) {
             using IUnitOfWork uow = p_unitOfWorkFactory.Create();
 
             using IRepository<TEntity> repo = uow.GetRepository<TEntity>();
 
-            TEntity? entites = repo.GetById(id, IncludesNavigationPathDetails);
+            TEntity? entity = repo.GetById(id, IncludesNavigationPathDetails);
 
-            return p_mapper.Map<TDetailModel>(entites);
+            if (entity is null)
+                return null;
```

Comment:
I would avoid placing the expression on the newline without braces, either leave it on single line or add braces

Reply:
fixed